tinygrad/device.py

```diff
@@ -48,13 +48,15 @@ class BufferOptions:
   cpu_access: bool = False
   host: bool = False
   nolru: bool = False
+  wgpu_bool: bool = False
```

Comment:
final solution cannot have these, could be a rewrite rule on store/load dtype

Reply:
Already have rewrite rules for bool. This is for a workaround that there are no bool buffers in webgpu, you have to create them as uint32. This hack is from the original pr. I have another idea for it.

Reply:
As discussed during the meeting this can stay for now.